src/app/lib/views/browser/list.js

```diff
@@ -273,7 +280,7 @@
                     '</div>' +
                 '</div>';
 
-            var ghosts = '<div class="ghost"></div>'.repeat(10);
+            var ghosts = '<div class="ghost"></div>'.repeat(this.itemsPerRow()*2);
```

Comment:
so i guess the *2 is to support not having glitches when resizeing from a much lower resolution, but that assumes that our max res is not bigger than 2x our smallest res...

i'm not sure it's a safe bet to take ? the correct value should be screensize(), but then what happens if we move the app to another screen that is bigger ?